api/internal/handler/rest/userHandler.go

```diff
@@ -1,33 +1,43 @@
 package handler
 
 import (
-	"backend/constants"
-	dbrepo "backend/infrastructure/repository/dbRepo"
-	"backend/utils"
+	"backend/api/internal/controller"
+	"backend/api/pkg/constants"
+	"backend/api/pkg/utils"
```

Comment:
```suggestion
	""errors"
	"fmt"
	"net/http"

	"backend/api/internal/controller"
	"backend/api/pkg/constants"
	"backend/api/pkg/utils"
	"github.com/mcnijman/go-emailaddress"
```
the idea when I said filter & sorting libs by groups like this:
group 1. contains internal go lib
group 2. contains installed libs by go get and internal path where directories called